tests/s3/versioning/test_get_put_object_tagging.py

```diff
@@ -0,0 +1,273 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+#
+# Copyright (c) 2022 Seagate Technology LLC and/or its Affiliates
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as published
+# by the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU Affero General Public License for more details.
+# You should have received a copy of the GNU Affero General Public License
+# along with this program. If not, see <https://www.gnu.org/licenses/>.
+#
+# For any questions about this software or licensing,
+# please email opensource@seagate.com or cortx-questions@seagate.com.
+#
+
+"""Test module for GET PUT Object Tagging"""
+
+import logging
+import os
+import time
+
+import pytest
+
+from commons.ct_fail_on import CTFailOn
+from commons.errorcodes import error_handler
+from commons.params import TEST_DATA_FOLDER
+from commons.utils import assert_utils
+from commons.utils.system_utils import create_file, path_exists, remove_file
+from commons.utils.system_utils import make_dirs, remove_dirs
+from config.s3 import S3_CFG
+from libs.s3.s3_tagging_test_lib import S3TaggingTestLib
+from libs.s3.s3_test_lib import S3TestLib
+from libs.s3.s3_versioning_common_test_lib import put_object_tagging
```

Comment:
can we import common lib as may be cmn_lib and avoid importing multiple functions?
same goes for system utils